xcube_geodb/core/message.py

```diff
@@ -0,0 +1,16 @@
+class Message:
+    def __init__(self, message: str):
+        self._message = message
+
+    @property
+    def message(self):
```

Comment:
A class named `Message` should not have a property with the same name. Consider `text` or `content`. Add type annotations.